appveyor.yml

```diff
@@ -20,6 +20,8 @@ before_deploy:
   - cp OpenRA.Game/OpenRA.ico .
   - appveyor DownloadFile "http://downloads.sourceforge.net/project/nsis/NSIS 2/2.46/nsis-2.46-setup.exe" -FileName nsissetup.exe
   - nsissetup /S /D=%NSIS_ROOT%
+  - appveyor DownloadFile "http://nsis.sourceforge.net/mediawiki/images/archive/1/18/20140806212030!NsProcess.zip" -FileName NsProcess.zip

```

Comment:
Download link is from here: http://nsis.sourceforge.net/NsProcess_plugin. This seems to be how things are done in the NSIS world. That link is probably as stable as they get there.